pages/design-tokens/color/state-tokens.md

```diff
@@ -5,6 +5,10 @@ title: State color tokens
 category: Design tokens
 lead: USWDS state colors tokens provide flexible, accessible, customizable colors for warning, error, and other states
 type: docs
+redirect_from:
+  - /components/colors/state/
+  - /colors/state/
+  - /color/state/
```

Comment:
Since we're redirecting from `/colors/` and `/color/`, should we do the same from the components path?

```suggestion
redirect_from:
  - /components/colors/state/
  - /components/color/state/
  - /colors/state/
  - /color/state/
```

Reply:
Not seeing a heavy use of color tokens in analytics, so I think it's okay to go without it (for now).